### PR TITLE
Add real meeting recap

### PR DESCRIPTION
**Pull request by paked**

### CODE REVIEW

File: case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+of my coleads to act as a middleman.
+
+## Conclusion
+
+I really want to look at getting our group to do the [Ludum
+Dare](http://ludumdare.com) game development jam again. As the last group which
+I worked through that with had really positive thoughts about it (and had a
+boat load of fun!). We are definitely making process and everyone seems very
+excited.
+
```
Comment (MaxWofford) [inline case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md]: @zachlatta Let's add a check for this.

Comment (paked) [inline case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md]: @MaxWofford a check for what?

Comment (MaxWofford) [inline case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md]: @paked There's an extra newline at the end of the file.

File: case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# First Meeting
+
+We had our first meeting today under the new reformed club banner. It was
+really awesome to see some new faces who were interested in coding at our
+school. Our year range currently only consists of the year below me, however I
+am going to bring in some of the younger students which I have previously
+connected with.  Unfortunately we had to cut the meeting short due to starting
```
Comment (MaxWofford) [inline case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md]: Would you please remove the double space from this line?

Comment (paked) [inline case_studies/glenaeon_rudolf_steiner_school/2015-11-04_first_meeting/recap.md]: @maxwofford pushed

**Pull request merged**